web/template/you_cannot_sign_your_lpa_yet.gohtml

```diff
@@ -0,0 +1,54 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "youCannotSignYourLpaYet" }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{ template "warning-banner" (warning .App "actorsMustBeOver18WarningBanner") }}
+
+            <h1 class="govuk-heading-xl">{{ tr .App "youCannotSignYourLpaYet" }}</h1>
+
+            {{ $actorsData := .Donor.Under18ActorDetails }}
+            {{ $from := printf "%s%s" $.App.Page "#f-date-of-birth" }}
+
+            {{ if eq 1 (len $actorsData) }}
+                {{ trHtml .App "singularActorYouCannotSignYourLpaYetContent" }}
+                {{ $from = .App.Paths.TaskList.Format .App.LpaID }}
```

Comment:
why change this in the single case?

Reply:
and if you don't need to change this you could use `{"one": ..., "other": ...}` for the content instead of an `if` here

Reply:
I was trying my best not to have to write a handler here so I hacked in some redirect logic for cases where the donor changes DOBs or removes actors and lands back on this page with 0 actors. Now I'm typing this out though it doesn't account for changing the DOB to something that is still under 18 so it probably makes sense to put a simple handler in to check this for redirects.

And cool, I'd forgotten about the pluralize feature.